.github/actions/setup-kind/action.yml

```diff
@@ -58,3 +67,17 @@ runs:
           --for=condition=ready pod \
           --selector=app.kubernetes.io/component=controller \
           --timeout=90s
+
+    - name: Set nginx ingress config
+      shell: bash
+      run: |
+        kubectl -n ingress-nginx patch cm ingress-nginx-controller \
+          -p '{"data": {${{inputs.ingress-configmap-patch}}}}'
```

Comment:
this should be optional as it's acs specific.

maybe it's better to have a new input with three states `off`, `allow-snippet-annotations`, `annotations-risk-level-critical`

it needs to be done before the previous wait step